app/session_test.go

```diff
@@ -319,41 +319,51 @@ func TestApp_ExtendExpiryIfNeeded(t *testing.T) {
 		require.False(t, session.IsExpired())
 	})
 
-	var tests = []struct {
-		name    string
-		session *model.Session
-	}{
-		{name: "mobile", session: &model.Session{UserId: model.NewId(), DeviceId: model.NewId(), Token: model.NewId()}},
-		{name: "SSO", session: &model.Session{UserId: model.NewId(), IsOAuth: true, Token: model.NewId()}},
-		{name: "web/LDAP", session: &model.Session{UserId: model.NewId(), Token: model.NewId()}},
-	}
-	for _, test := range tests {
-		t.Run(fmt.Sprintf("%s session beyond threshold should update ExpiresAt", test.name), func(t *testing.T) {
-			session, err := th.App.CreateSession(test.session)
-			require.Nil(t, err)
-
-			expires := model.GetMillis() + th.App.GetSessionLengthInMillis(session) - hourMillis
-			session.ExpiresAt = expires
-
-			ok := th.App.ExtendSessionExpiryIfNeeded(session)
-
-			require.True(t, ok)
-			require.Greater(t, session.ExpiresAt, expires)
-			require.False(t, session.IsExpired())
-
-			// check cache was updated
-			var cachedSession *model.Session
-			errGet := th.App.Srv().sessionCache.Get(session.Token, &cachedSession)
-			require.NoError(t, errGet)
-			require.Equal(t, session.ExpiresAt, cachedSession.ExpiresAt)
-
-			// check database was updated.
-			storedSession, nErr := th.App.Srv().Store.Session().Get(context.Background(), session.Token)
-			require.NoError(t, nErr)
-			require.Equal(t, session.ExpiresAt, storedSession.ExpiresAt)
-		})
-	}
+	for i := 0; i <= 1; i++ {
```

Comment:
Nit: This confused me. Maybe including this as a parameter in the tests would be more clear?